README.md

```diff
@@ -64,7 +64,7 @@ comes with everything you need to start your research, including:
 
 ## Quick install
 
-You will need Python 3.6 or later and a working [JAX](https://github.com/google/jax/blob/main/README.md)
```

Comment:
Perhaps:

"...3.7 and above, and a working JAX installation..."

or

"...3.7-3.10, and a working JAX installation..."

@IvyZX @cgarciae